website/docs/docs/cloud/dbt-cloud-ide/lint-format.md

```diff
@@ -183,6 +183,23 @@ To format your Python code, dbt Cloud integrates with [Black](https://black.read
 
 ## FAQs
 
+<details>
+<summary>When should I use SQLFluff and when should I use sqlfmt?</summary>
+
+SQLFluff and sqlfmt are both tools used for formatting SQL code, but there are some differences that may make one preferable to the other depending on your use case. <br />
+
+SQLFluff is a SQL code linter and formatter. This means that it analyzes your code to identify potential issues and bugs, and follows coding standards. It also formats your code according to a set of rules, which are [customizable](#customize-linting), to ensure consistent coding practices. You can also use SQLFluff to keep your SQL code well-formatted and follow styling best practices. <br />
+
+sqlfmt is a SQL code formatter. This means it automatically formats your SQL code according to a set of formatting rules that aren't customizable. It focuses solely on the appearance and layout of the code, which helps ensure consistent indentation, line breaks, and spacing. sqlfmt doesn't analyze your code for errors or bugs and doesn't look at coding issues beyond code formatting. <br />
+
+You can use either SQLFluff or sqlfmt depending on your preference and what works best for you:
+
+- Use SQLFluff to have your code linted and formatted (meaning analyze fix your code for errors/bugs, and format your styling). It allows you the flexibility to customize your own rules.
+
+- Use sqlfmt to only have your code well-formatted, without analyzing your code for errors and bugs. You can't customize rules with it.
```

Comment:
Maybe add something about how sqlfmt provides this "out of the box" and that would be the benefit. 

> have your code well-formatted, as the default standard, without the need to customize it

Reply:
good shout @Victoriapm ! ok reworded it to the below, wdyt?

>Use sqlfmt to only have your code well-formatted without analyzing it for errors and bugs. You can use sqlfmt out of the box, making it convenient to use right away without having to configure it.